libraries/abstractions/common_io/test/test_iot_rtc.c

```diff
@@ -712,79 +686,83 @@ TEST( TEST_IOT_RTC, AFQP_IotRtcSetAlarmInvalid )
     int32_t lRetVal;
 
     /* Open rtc to initialize hardware. */
-    if( gIotRtcHandle == NULL )
-    {
-        xRtcHandle = iot_rtc_open( 0 );
-        TEST_ASSERT_NOT_EQUAL( NULL, xRtcHandle );
-    }
-    else
-    {
-        xRtcHandle = gIotRtcHandle;
-    }
+    xRtcHandle = iot_rtc_open( uctestIotRTCInstanceIdx );
+    TEST_ASSERT_NOT_EQUAL( NULL, xRtcHandle );
 
+    xSetDateTime.ucWday = testIotRTC_DEFAULT_CURRENT_WEEK_DAY;
     xSetDateTime.usYear = testIotRTC_DEFAULT_CURRENT_YEAR;
     xSetDateTime.ucMonth = testIotRTC_DEFAULT_CURRENT_MONTH;
     xSetDateTime.ucDay = testIotRTC_DEFAULT_CURRENT_DAY;
     xSetDateTime.ucHour = testIotRTC_DEFAULT_CURRENT_HOUR;
     xSetDateTime.ucMinute = testIotRTC_DEFAULT_CURRENT_MINUTE;
-    xSetDateTime.ucSecond = testIotRTC_DEFAULT_CURRENT_SECOND;
+    xSetDateTime.ucSecond = testIotRTC_DEFAULT_CURRENT_SECOND + testIotRTC_DEFAULT_ALARM_TIME;
```

Comment:
Why is the DEFAULT_ALARM_TIME added here to the default time being set? #722 may not return error if its the current second.

Reply:
The reason is due to the fact that testIotRTC_DEFAULT_CURRENT_SECOND is zero. The original setup would subtract 5 from 0 in a uint8 causing a underflow. That said, it could be a good idea to bump it so that xSetDateTime.ucSecond > testIotRTC_DEFAULT_ALARM_TIME.

You agree with bumping it to say 2 *  testIotRTC_DEFAULT_ALARM_TIME? Alternatively, set testIotRTC_DEFAULT_CURRENT_SECOND > testIotRTC_DEFAULT_ALARM_TIME.

Reply:
Yes. Didnt realize that. I agree to either of the approach

Reply:
Updated.